design/ceph/ceph-config-via-cluster-crd.md

```diff
@@ -0,0 +1,53 @@
+# Ceph Config via Ceph Cluster CRD
+
+## Goal
+
+Adding an addition simpler way to configure the Ceph config options via the CephCluster CRD.
+The goal is not to replace the `rook-config-override` ConfigMap yet, but to reduce it's use to almost zeor for 99% of the cases.
+
+## Current Solution
+
+Users need to update the `rook-config-override` and add their custom options, e.g., for setting a custom OSD scrubbing schedule ([Ceph OSD Scrubbing config options](https://docs.ceph.com/en/latest/rados/configuration/osd-config-ref/#scrubbing)).
+
+## Proposed Solution
+
+Adding a new config structure to the CephCluster CRD under `.spec.storage` named `scrubbing:`.
+
+The config structure will be structured in a way of `Target->Options`.
+
+```yaml
+spec:
+  # [...]
+  config:
```

Comment:
What about `cephConfig`? Then perhaps it is more clear these are just being passed on to ceph? `config` is also used a few other places in the `cephCluster` CR under `storage`, so it could be easily confused with those settings.
```suggestion
  cephConfig:
```